extensions/applicationinsights-perfmarkmeasure-js/scripts/listCdnVersions.ps1

```diff
@@ -669,17 +669,17 @@ $files = New-Object 'system.collections.generic.dictionary[string, system.collec
 
 # Get the beta files
 if ([string]::IsNullOrWhiteSpace($container) -eq $true -or $container -eq "beta") {
-    GetVersionFiles $files "beta/ext" "ai.clck."
+    GetVersionFiles $files "beta/ext" "ai.prfmm-mgr."
```

Comment:
Just fixing up the publishing script as this was a copy of the click analytics scripts -- but I missed updating with the correct name.
I've also tested these now against a real test cdn storage account.